plugins/workbench-resources/src/components/HelpAndSupport.svelte

```diff
@@ -0,0 +1,241 @@
+<script lang="ts">
+  import {
+    Button,
+    closePopup,
+    getCurrentLocation,
+    Icon,
+    IconArrowLeft,
+    Label,
+    ListView,
+    navigate,
+    Scroller
+  } from '@hcengineering/ui'
+  import setting, { settingId } from '@hcengineering/setting'
+  import view, { Action, ActionCategory } from '@hcengineering/view'
+  import { getClient } from '@hcengineering/presentation'
+  import RightArrowIcon from './icons/Collapsed.svelte'
+  import KeyboardIcon from './icons/Keyboard.svelte'
+  import DocumentationIcon from './icons/Documentation.svelte'
+  import workbench from '../plugin'
+
+  let shortcuts = false
+  let actions: Action[] = []
+  let categories: ActionCategory[] = []
+
+  const client = getClient()
+
+  function navigateToSettings () {
+    closePopup()
+    const loc = getCurrentLocation()
+    loc.path[2] = loc.path[3] = settingId
+    loc.path.length = 4
+    navigate(loc)
+  }
+
+  function formatKey (key: string): string[][] {
```

Comment:
@muhtimur  It's do some utility method to combine one from view-resources/ActionsPopup.svelte with this one, in next PR.

Reply:
Ok